python/README.md

```diff
@@ -0,0 +1,45 @@
+# Python Data Access Utilities
+
+This package provides utility functions for accessing our internal data infrastructure, including Azure Blob Storage and PostgreSQL databases.
+
+## Directory Structure
+
+```
+.
+├── requirements.txt        # Project dependencies
+├── setup.py                # Package configuration
+├── pyproject.toml          # Package configuration
+├── src/                    # Source code for utilities 
+└── examples/               # Usage examples
+```
+
+## Developer Setup
+
+1. Create and activate a virtual environment:
+```bash
+python -m venv venv
+source venv/bin/activate  # On Windows: venv\Scripts\activate
+```
```

Comment:
The equivalent for `pyenv virtualenv` would be:

```bash
pyenv virtualenv <PYTHON_VERSION> virtualenv <VIRTUALENV_NAME>
pyenv local <VIRTUALENV_NAME>
pyenv activatate <VIRTUALENV_NAME>
```

For example:

```bash
pyenv virtualenv 3.11.4 virtualenv ds-my-repository
pyenv local ds-my-repository
pyenv activate ds-my-repository
```

Optionally you can add `eval "$(pyenv virtualenv-init -)"` to your `.zshrc` to automatically activate virtualenvs when you're in the directory.

Reply:
Added!

Reply:
Ah sorry just saw my typo in `pyenv activatate`...